src/i18n/config.json

```diff
@@ -108,13 +108,13 @@
     "enabled": false
   },
   {
-    "code": "pt-br",
-    "localName": "Português do Brasil",
-    "name": "Brazilian Portuguese",
+    "code": "pt",
```

Comment:
Is the intention to implement `pt-PT`? If so, I don't think wiping out the Brazilian key is correct, but you'd add an additional one. The existing translations on the `.org` are Brazilian Portuguese https://crowdin.com/project/nodejs-website or at least presented that way. The API docs are also split that way https://crowdin.com/project/nodejs

Reply:
> Is the intention to implement `pt-PT`? If so, I don't think wiping out the Brazilian key is correct, but you'd add an additional one. The existing translations on the `.org` are Brazilian Portuguese https://crowdin.com/project/nodejs-website or at least presented that way. The API docs are also split that way https://crowdin.com/project/nodejs

@nschonni do you think that is better to just not touch the Brazillian Portuguese and just add specifications for the Portuguese I want to contribute to?

I would like to say that is not so bad having two or more variations of Portuguese. I am doing the updates and adjustments now.

Reply:
Sorry, can't tell you why, but I've seen this on many tech projects. I would guess due to the size of the Brazilian developer community. Chinese dialects would be the other commonly split one without going fully l10n vs. i18n

Reply:
Like @nschonni here, I don't think you should remove the Brazilian Portuguese locale config, rather add one for the generic Portuguese language.